tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -64,6 +64,11 @@ def resource_id(self):
         spliced = self.relative_id.split("attributeMappings/")[1]
         return spliced
 
+    def delete(self, url, client):
+        delete_url = url + "/" + self.resource_id
```

Comment:
This is the same as `self.relative_id`

Reply:
self.relative_Id = "projects/4/attributeMappings/19629-12"
delete_url ="http://localhost:9100/api/versioned/v1/projects/4/attributeMappings/19629-12"

Reply:
`delete_url` should just need to be `projects/4/attributeMappings/19629-12` because the client will have a base path of `http://localhost:9100/api/versioned/v1`